flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotProvider.java

```diff
@@ -102,20 +91,6 @@ default void start(ComponentMainThreadExecutor mainThreadExecutor) {
 			allocationTimeout);
 	}
 
-	/**
-	 * Allocates a bulk of physical slots. The allocation will be completed
-	 * normally only when all the requests are fulfilled.
-	 *
-	 * @param physicalSlotRequests requests for physical slots
-	 * @param timeout indicating how long it is accepted that the slot requests can be unfulfillable
-	 * @return future of the results of slot requests
-	 */
-	default CompletableFuture<Collection<PhysicalSlotRequest.Result>> allocatePhysicalSlots(
-		Collection<PhysicalSlotRequest> physicalSlotRequests,
-		Time timeout) {
-		throw new UnsupportedOperationException("Not properly implemented.");
-	}
-
 	/**
 	 * Cancels the slot request with the given {@link SlotRequestId} and {@link SlotSharingGroupId}.
 	 *
```

Comment:
Can default `void cancelSlotRequest(SlotRequestId slotRequestId, Throwable cause)` be removed?